content/applications/marketing/surveys/create.rst

```diff
@@ -1,224 +1,331 @@
 
-.. image:: create/questions-description-options.png
-   :align: center
-   :alt: Various tabs that can be found on the survey template page.
+Beneath those radio survey type options is a blank field in which a name for the survey **must** be
+entered.
+
+Below the survey name field, is the :guilabel:`Responsible` field. Choose a user from the
+drop-down menu to be in charge of the survey. By default, the user who initially created the survey
+is selected as the default :guilabel:`Responsible`.
+
+To the right of those fields, and above the tabs, is the option to add a background image,
+represented by a :guilabel:`📷 (camera)` icon. When clicked, the option to upload an image becomes
+available. This image would be used as the background image for the entire survey. This is **not** a
+required option.
+
+Below those fields and options are four tabs: :guilabel:`Questions`, :guilabel:`Options`,
+:guilabel:`Description`, and :guilabel:`End Message`.
 
 Questions tab
-=============
+-------------
+
+View, access, add, and/or delete questions and sections to the survey in the :guilabel:`Questions`
+tab.
 
-Add questions and sections to the survey in the :guilabel:`Questions` tab. A section divides the
-survey into parts in order to visually group similar questions together. To make a section, click
-:guilabel:`Add a section` and type in a section name. Then, add questions or drag and drop
-questions into the divided sections.
+By default, two columns are present in the :guilabel:`Questions` tab: :guilabel:`Title` (i.e. the
+question) and :guilabel:`Question Type`.
 
-Clicking :guilabel:`Add a question` opens the :guilabel:`Create Sections and Questions` pop-up to
-create and customize the survey question.
+If the :guilabel:`Randomized per Section` option is enabled in the :guilabel:`Options` tab of the
+survey form, a column titled, :guilabel:`# Questions Randomly Picked` appears in the
+:guilabel:`Questions` tab.
 
-.. image:: create/survey-question-pop-up.png
+Indicate whether questions require a mandatory answer by clicking the :guilabel:`(slider)` icon to
+the far-right of the column titles. Then, select :guilabel:`Mandatory Answer` from the drop-down
+menu to reveal the :guilabel:`Mandatory Answer` column in the :guilabel:`Questions` tab.
+
+.. image:: create/mandatory-answer-dropdown.png
    :align: center
-   :alt: The survey question pop-up window.
+   :alt: The slider drop-down menu with the Mandatory Answer option selected in Odoo Surveys.
 
-Create questions
-----------------
+Add a question
+~~~~~~~~~~~~~~
 
-In the :guilabel:`Create Sections and Questions` pop-up, type the question in the
-:guilabel:`Question` field. Then, choose the :guilabel:`Question Type`. A preview of how the
-question type looks is shown in the preview window.
+To add a question to a survey, click :guilabel:`Add a question` in the :guilabel:`Questions` tab,
+and proceed to fill out the :guilabel:`Create Sections and Questions` pop-up window that appears.
```

Comment:
This is a modal UI element
```suggestion
and proceed to fill out the :guilabel:`Create Sections and Questions` modal that appears.
```

Reply:
I know this comment was *before* we decided to stick with "pop-up..." for the novice readers...so, I'm going to disregard. Appreciate the suggestion, though 👍